nexus/types/src/inventory.rs

```diff
@@ -364,13 +363,15 @@ impl IntoRotPage for gateway_client::types::RotCfpa {
 /// the disk is being actively managed by the control plane.
 #[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
 pub struct PhysicalDisk {
+    // TODO: should this just be InventoryDisk? Do we need a separation between
```

Comment:
I don't actually think we do need the separation since the types are structurally the same. I can't recall if a `PhysicalDisk` always shows up in inventory though.

Reply:
This PR is big enough already and addressing this isn't hugely impactful, so I'll just leave a comment here for now.